code/global.dm

```diff
@@ -22,8 +22,10 @@
 #define R_HOST			(1<<15)
 #define R_PROFILER		(1<<16)
 #define R_NOLOCK		(1<<17)
+#define R_EVENT			(1<<18)
 
-#define R_EVERYTHING	((1<<18)-1) //the sum of all other rank permissions
+#define R_EVERYTHING	((1<<19)-1) //the sum of all other rank permissions
+#define G_DEVELOPER		(1<<20)
```

Comment:
this should probably be R_DEVELOPER as well, and put it beneath (lower bit value) the R_EVERYTHING thing

Reply:
I made it G as it's not really a RIGHT so much as a GROUP. Setting it under everything would likely stop managers being split differently.

Reply:
if you want to create groups, the entire code needs to be refactored. Making it a flag won't do anything but contribute to writing bad code

Reply:
Tied to profiler as admins have R_DEBUG